goi18n/constants_command.go

```diff
@@ -34,6 +41,7 @@ var constTemplate = template.Must(template.New("").Parse(`// DON'T CHANGE THIS F
 
 package {{.PackageName}}
 {{range .Constants}}
+// {{.Name}} {{.Comment}}
```

Comment:
this model will generate name and comment on different lines, but golint demands name and comment on the same line
```go
// Name Comment
```